### PR TITLE
Update off.xml

### PR DESCRIPTION
**Pull request by AurelioDeRosa**
Fix issue #384.

### CODE REVIEW

**Comment by arthurvr** [comment]
This PR adds a `<signature>`, but I'd like to see a little more. A more in-depth explanation, an example maybe?

**Comment by AurelioDeRosa** [comment]
@arthurvr I didn't include an example because the `one()` method has one that uses this signature and I didn't want to replicate it.

File: entries/off.xml [hunk 1]
```diff
@@ -26,6 +26,12 @@
   </signature>
   <signature>
     <added>1.7</added>
+    <argument name="event" type="Event">
+      <desc>A <a href="/category/events/event-object/"><code>jQuery.Event</code></a> object.</desc>
+    </argument>
+  </signature>
+  <signature>
+    <added>1.7</added>
```
Comment (dmethvin) [inline entries/off.xml]: This signature was around earlier than 1.7, I had to keep it working when I refactored the code in 1.7. I'll look for the right version.

Comment (dmethvin) [inline entries/off.xml]: Oh but not in `.off()` :smile: Carry on.

**Comment by arthurvr** [comment]
LGTM. Shall I merge?

**Comment by kswedberg** [comment]
@arthurvr :+1:

**Pull request closed without merging**